examples/oidc/css-api/nextjs/src/pages/api/integrations/[integrationID]/[environment]/users/[username]/roles/index.js

```diff
@@ -0,0 +1,29 @@
+import { fetchCssApiCredentials, apiErrorHandler, handleStatusError } from '@/utils'
```

Comment:
is handleStatusError used in this method?

Reply:
It is not! Good catch I'll remove that